graphqlapi/src/main/java/io/stargate/graphql/schema/types/scalars/BigIntCoercing.java

```diff
@@ -0,0 +1,70 @@
+package io.stargate.graphql.schema.types.scalars;
+
+import graphql.language.IntValue;
+import graphql.language.StringValue;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import java.math.BigInteger;
+
+/**
+ * Handles type conversions for CQL {@code bigint}.
+ *
+ * <p>Our default mapping is String: client-side Javascript libraries cannot safely handle integers
+ * outside of the range [-2^53, 2^53-1], so we can't return a Long.
```

Comment:
Verified with the playground UI: I was getting weird results when reading back a row that contains `Long.MAX_VALUE`.
I hadn't realized that query results are pure JSON, @jorgebay's explanations make more sense now.